python/events/tests/test_interception.py

```diff
@@ -0,0 +1,97 @@
+"""Tests functionality of the interception module."""
+import pytest
+import hydrotools.events.models.interception as itc
+import numpy as np
+
+def test_compute_interception() -> None:
+    """Tests basic functionality of top level method of interception module.
+    Also tests that the method correctly raises with bad inputs. Specifically,
+    if the gross rainfall, average rainfall rate, or canopy fraction is 0.0,
+    the method should raise an AssertionError."""
```

Comment:
This is true only if python optimization isn't enabled `python -O *.py` will turn off assertions.

Reply:
Generally, this won't be an issue for `test_*` modules because those are meant to be run by `pytest`. However, the `interception` module proper should definitely have "Raises" sections in the docstrings. I neglected to do that. I'll add this note.

Reply:
With respect to the use of `pytest`, it's probably worth updating `CONTRIBUTING.md` with this note about optimizations.